src/sentry/static/sentry/app/views/organizationIntegrations/integrationRequest/RequestIntegrationModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import {addErrorMessage, addSuccessMessage} from 'app/actionCreators/indicator';
+import {ModalRenderProps} from 'app/actionCreators/modal';
+import AsyncComponent from 'app/components/asyncComponent';
+import Button from 'app/components/button';
+import {t} from 'app/locale';
+import TextareaField from 'app/views/settings/components/forms/textareaField';
+import TextBlock from 'app/views/settings/components/text/textBlock';
+
+import {RequestIntegrationProps} from '.';
+
+type Props = {
+  onSuccess: () => void;
+} & RequestIntegrationProps &
```

Comment:
@mgaeta Rather than making a separate type you import like this, the practice we follow is just using the type of the component. See `src/sentry/static/sentry/app/components/roleSelectControl.tsx` for an example of how that is done. Something like `RequestIntegrationButton['props']`.